support-frontend/assets/pages/[countryGroupId]/components/oneTimeCheckoutComponent.tsx

```diff
@@ -423,6 +419,13 @@ export function OneTimeCheckoutComponent({
 						);
 					}
 				} else {
+					const stripePaymentMethod: StripePaymentMethod =
+						paymentMethod === 'StripeExpressCheckoutElement'
+							? stripeExpressCheckoutPaymentType === 'apple_pay'
+								? 'StripeApplePay'
+								: 'StripePaymentRequestButton'
+							: 'StripeCheckout';
```

Comment:
Should this be `Stripe`? Wonder whether this is leaking down from line 307.

Reply:
It's 'StripeCheckout' in the [generic checkout](https://github.com/guardian/support-frontend/blob/cf852b87a6734ed91f85145eaa87973fb8697b48/support-frontend/assets/pages/%5BcountryGroupId%5D/components/checkoutComponent.tsx#L581) and the old code. And it worked when tested in CODE/dev